crest/Assets/Crest/Crest/Scripts/LodData/LodDataMgrSeaFloorDepth.cs

```diff
@@ -16,11 +16,13 @@ namespace Crest
     /// </summary>
     public class LodDataMgrSeaFloorDepth : LodDataMgr
     {
+        internal const float k_DepthBaseline = 1_000f;
```

Comment:
may be worth also referencing the constant in the shader code from here (you have already referenced in the other direction)

Reply:
Done. Thanks.